modules/core/src/main/scala/fs2/kafka/CommittableOffset.scala

```diff
@@ -131,4 +130,12 @@ object CommittableOffset {
 
   implicit def committableOffsetShow[F[_]]: Show[CommittableOffset[F]] =
     Show.fromToString
+
+  implicit def committableOffsetEq[F[_]]: Eq[CommittableOffset[F]] =
+    Eq.instance {
+      case (l, r) =>
+        l.topicPartition == r.topicPartition &&
+          l.offsetAndMetadata == r.offsetAndMetadata &&
+          l.consumerGroupId == r.consumerGroupId
+    }
```

Comment:
I'm not sure that this is 100% accurate because `CommittableOffset` also has a `commit: F[Unit]`. In my implementation, I just ignore this field.

Reply:
Yeah, that makes me slightly nervous. Do you have any thoughts on this @vlovgr?

Reply:
I think this is the reason why these instances haven't been added before. In practice though, we should never have two consumers with the same (group-id, topic-partition, offset), so not sure how this would be an issue. 🤷‍♂️

Reply:
I think, in practice, this will not be a problem. 

But in terms of API, it feels a bit weird. `CommittableOffset` feels like pure data, but it contains `def commit: F[Unit]`, which is not data, but a behavior. Maybe it would be better if `CommittableOffset` will not have `commit` method? Maybe it would be more natural to have it on `KafkaConsumer` (like `def commit(committableOffset: CommitableOffset): F[Unit]`)?

Of course, this change will not be backward compatible, and I think it not worth it. Maybe, someday, in some major update :)

Reply:
Would you mind opening an issue about this and we can schedule it for 2.0? :)

Reply:
Ok: https://github.com/fd4s/fs2-kafka/issues/462